src/util/stringFormatting.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Function that accepts string and replaces underscores with spaces
```

Comment:
1. please don't start describing the function based on what it *accepts*: this is evident by the params
2. example: `Replace all underscore string ocurrances with space, ie: '_' -> ' '